baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -80,6 +87,48 @@ def stop(self) -> None:
         # `should_stop` is an attribute of `ConsumerMixin`
         self.should_stop = True
 
+    def __del__(self) -> None:
+        if not self.should_stop:
+            self.stop()
```

Comment:
🔕  I'm not sure if this addition is really necessary, shouldn't the `ConsumerMixin` already handle this?  It also doesn't hurt.

Reply:
The `ConsumerMixin` doesn't handle it, but yes, it should! I've moved in there.